.babelrc

```diff
@@ -5,14 +5,18 @@
   ],
   "plugins": [
       "transform-flow-comments",
-      "syntax-flow",
-      "transform-runtime"
+      "syntax-flow"
   ],
   "env": {
     "test": {
       "plugins": [
         "rewire"
       ]
+    },
+    "production": {
```

Comment:
I think we just need NODE_ENV or BABEL_ENV to be set to production with the build script